src/base/Ownable.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: MIT
```

Comment:
would using this and crediting solmate be even smaller in size? https://github.com/transmissions11/solmate/blob/main/src/auth/Owned.sol

Reply:
Tried , bigger than OZ version.
https://github.com/pancakeswap/pancake-v4-core/commit/58a73576b53aadb7ebab05354f57e6d073535ec7#diff-adfe825c0db3d62e6991070eb98f450a367d176e61611151bcb1556bfb5e8780

Reply:
thanks! 🙏 marking this as resolved